CMakeLists.txt

```diff
@@ -45,11 +47,24 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-set(TRITON_TENSORFLOW_LIBNAME "libtensorflow_trtis.so")
+TRITON_TENSORFLOW_DOCKER_BUILD=OFF
+if(TRITON_TENSORFLOW_LIB_PATHS MATCHES "")
+  TRITON_TENSORFLOW_DOCKER_BUILD=ON
+endif()
+
+if (${TRITON_TENSORFLOW_DOCKER_BUILD})
+  set(TRITON_TENSORFLOW_LIBNAME "libtensorflow_trtis.so")
+else()
+  set(TRITON_TENSORFLOW_LDFLAGS "-ltensorflow_trtis")	
+endif() # TRITON_TENSORFLOW_DOCKER_BUILD
 set(TRITON_TENSORFLOW_BACKEND_LIBNAME triton_tensorflow1)
 set(TRITON_TENSORFLOW_BACKEND_INSTALLDIR ${CMAKE_INSTALL_PREFIX}/backends/tensorflow1)
 if (${TRITON_TENSORFLOW_VERSION} EQUAL "2")
```

Comment:
Can you nest the "2" checks into the above conditional to keep the related code together.. at lines 56 and 58.

Reply:
Done.